server/middlewares/authenticate.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken')
+
+const authenticate = (req, res, next) => {
+    try{
+      const token = req.headers.token
+      const decoded = jwt.verify(token, process.env.SECRET )
```

Comment:
verify pindahin ke helpers juga
```suggestion
const verify = require('../helpers/jwt')

const authenticate = (req, res, next) => {
    try{
      const token = req.headers.token
      const decoded = verify(token, process.env.SECRET)
```